constd/main.go

```diff
@@ -98,7 +98,6 @@ func (c *constd) reconcileAtmo(errchan chan error) {
 			"ATMO_HTTP_PORT="+atmoPort,
 			"ATMO_CONTROL_PLANE="+c.config.controlPlane,
 			"ATMO_ENV_TOKEN="+c.config.envToken,
-			"ATMO_HEADLESS=true",
```

Comment:
Before merging this, please change the 'c.logger.Info("reconciling", runnable.FQFN)` line down below to `c.logger.Debug`... the log output for constd is insufferable :P

Reply:
Fixed in https://github.com/suborbital/sat/pull/42/commits/e81b196ff9a3e4a491d58ad2bea77637c7cb1b5d